### PR TITLE
Update jupyter

### PR DESCRIPTION
**Pull request by annoyingllama**
I confirm that I have added:

- [x] A comment to every Dockerfile with information about target repository and tag (version).
- [x] Filled in readme on dockerhub and in this repo about how to build the image if some extra steps should be done.

update the jupyter base image to the most current tag 703d8b2dcb88

### CODE REVIEW

File: jupyter/Dockerfile
Comment (mpasa) [inline jupyter/Dockerfile:7]: In this doing something? Our spark version for data don't depend on Jupyter, but on the kernel mainly.

**Review by ValentinMouret — APPROVED** [review]
Nice labels :)

**Review by t-redactyl — APPROVED** [review]
Good that we're now the maintainer as well.

File: jupyter/Dockerfile
Comment (annoyingllama) [reply jupyter/Dockerfile:7]: not this just tells that the local installation on the image is 3.0.1 but it should be fine

Comment (annoyingllama) [reply jupyter/Dockerfile:7]: i put versions there so that in future you could simple do `docker image inspect ` to get some metadata on the image that might be useful

**Comment by annoyingllama** [comment]
I would prefer is we deploy and test the image from this branch before merging to master.

File: jupyter/Dockerfile
Comment (mpasa) [reply jupyter/Dockerfile:7]: 👍

**Comment by mpasa** [comment]
@ValentinMouret Can you help with the deploy? Or do you want data to help here?

**Review by mpasa — APPROVED** [review]
Looks good! And cleaner using Poetry

**Comment by mpasa** [comment]
@annoyingllama Can we merge this?

**Comment by annoyingllama** [comment]
this is now redundant as we have new jupyterhub on gcp closing this PR

**Pull request closed without merging**
